calico_node/filesystem/sbin/check_readiness

```diff
@@ -0,0 +1,51 @@
+#!/bin/sh
+not_ready_msg="calico/node is not ready"
+
+# Check felix readiness
+if [ "$FELIX_HEALTHENABLED" = "true" ]; then
+    /usr/bin/wget -T 5 localhost:9099/readiness -O - &> /dev/null
+    if [ $? -ge 400 ] || [ $? -lt 200 ]; then
+        echo "$not_ready_msg: felix is not ready"
+        exit 1
+    fi
+fi
+
+# Check bird peer status
+unestablished_peers()
+{
+    /bin/birdcl -s /var/run/calico/bird.ctl show protocols | tail -n +3 | while read -r line; do
+        protocol=`echo "$line" | awk '{print $2}'`
+        if [ "$protocol" != "BGP" ]; then
+            continue
+        fi
+
+        status=`echo $line | awk '{print $6}'`
+        if [ "$status" != "Established" ]; then
+            peer=`echo $line | awk '{print $1}'`
+            printf "$peer "
+        fi
+    done
+}
+# Check that bird is enabled.
```

Comment:
That's what this is doing. `/etc/service/enabled/bird` is a more explicit indication of whether bird is running or not, since `CALICO_NETWORKING_BACKEND`  has [some interesting wildcard factors](https://github.com/projectcalico/calico/blob/master/calico_node/filesystem/etc/rc.local#L50-L51) as to when it runs bird.

Reply:
Right, I see.

But...we still try to talk to bird below this to check its status, even when it isn't enabled, so I think there's still an issue

Reply:
I don't really understand the point about wildcard factors - if `CALICO_NETWORKING_BACKEND=bird` and the file `/etc/service/enabled/bird` doesn't exist, then something has gone wrong and and we should fail the readiness check... right?